projects/rero/ng-core/src/lib/record/search/record-search.component.ts

```diff
@@ -92,25 +61,20 @@ export class RecordSearchComponent implements OnInit, OnChanges {
   sort: string = null;
 
   /**
-   * Search query
+   * Current filters applied
    */
   @Input()
-  q = '';
+  aggFilters = [];
```

Comment:
why not aggregationFilters as the other components?

Reply:
changed in the whole project.